### PR TITLE
Update package

### PR DESCRIPTION
**Pull request by tiagotex**
Not sure about the `mix.lock`, should I use another command to generate it instead of the `mix deps.get`?

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -43,10 +43,10 @@ defmodule Bugsnex.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:httpoison, "~> 0.8.3"},
+      {:httpoison, "~> 0.11"},
```
Comment (PragTob) [inline mix.exs]: do we really need the new version?

File: mix.exs [hunk 2]
```diff
@@ -43,10 +43,10 @@ defmodule Bugsnex.Mixfile do
...
       {:poison, "~> 1.5 or ~> 2.0"}, # to be compatible with phoenix_ecto
-      {:plug, "~> 1.1"},
-      {:phoenix, "~> 1.1", only: :test},
+      {:plug, "~> 1.3"},
+      {:phoenix, "~> 1.2.1", only: :test},
```
Comment (PragTob) [inline mix.exs]: I don't think we need those new versions either

File: mix.exs [hunk 3]
```diff
@@ -43,10 +43,10 @@ defmodule Bugsnex.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:httpoison, "~> 0.8.3"},
+      {:httpoison, "~> 0.11"},
       {:poison, "~> 1.5 or ~> 2.0"}, # to be compatible with phoenix_ecto
```
Comment (PragTob) [inline mix.exs]: here we'd need a new version - or well we also want to allow poision 3.x so I guess `>= 1.5` might be finde

File: mix.exs [hunk 4]
```diff
@@ -43,8 +43,8 @@ defmodule Bugsnex.Mixfile do
   # Type "mix help deps" for more examples and options
   defp deps do
     [
-      {:httpoison, "~> 0.8.3"},
-      {:poison, "~> 1.5 or ~> 2.0"}, # to be compatible with phoenix_ecto
+      {:httpoison, "~> 0.11"},
+      {:poison, "~> 1.5 or ~> 2.0 or ~> 3.0"}, # to be compatible with phoenix_ecto
```
Comment (PragTob) [inline mix.exs]: imo this could just be `>= 1.5` couldn't it?

Comment (tiagotex) [inline mix.exs]: Fixed. Before I was getting an error because >= 1.5 returns an error.

**Pull request merged**